src/pages/index.tsx

```diff
@@ -32,10 +33,12 @@ export default function Home(): JSX.Element {
     <Layout
       title={`Hello from ${siteConfig.title}`}
       description="Description will go into a meta tag in <head />">
-      <HomepageHeader />
-      <main>
-        <HomepageFeatures />
-      </main>
+        <RootContext>
```

Comment:
this is gonna apply the `RootContext` to main page and not the whole application. 
please check this link https://docusaurus.io/docs/swizzling#wrapper-your-site-with-root
thanks thanks.